src/components/context/EditItemModalContext.js

```diff
@@ -14,6 +14,10 @@ import { ITEM_FORM_CONFIRM_BUTTON_ID } from '../../config/selectors';
 import { ITEM_TYPES } from '../../enums';
 import BaseItemForm from '../item/form/BaseItemForm';
 import DocumentForm from '../item/form/DocumentForm';
+import { isItemValid } from '../../utils/item';
+
+// time to be considered between 2 clicks for a double-click (https://en.wikipedia.org/wiki/Double-click#Speed_and_timing)
+const DOUBLE_CLICK_DELAY_MS = 500;
```

Comment:
Would it be possible to factor this logic in the graasp button (in ui)? (not to do immediately, but it might be a nice to apply this prevention on all our buttons)